### DISCUSSION TITLE
There needs to be a canonical ElectOS blueprint

### DISCUSSION THREAD

**Discussion by cwulfman** [ion-oset/developer-commons#15] [RFCs]
There is no obvious canonical blueprint of all of ElectOS's components -- at least none that is visible to developers.  Different development teams have different code names and different conceptions of the various pieces, based on white papers, presentation slides, and demos.  Design and development would benefit from a common reference point.

**Comment by stratofax** [comment]
Perhaps we can draft one in Mermaid? That will make it easy to update, an excellent addition to the code in this repo!

**Reply by cwulfman** [reply]
I'm down with that. But what if we were to begin by compiling a reference document with pointers to all the known expressions of the architecture?  E.g., [U.S. Digital Absentee Voting System - NVP Scoping Document Final](https://docs.google.com/document/d/1HUqDAnpbEQU5izkYOe0s5iSfcChC6VA9/edit), which seems to be informing the Tusk-funded development.

**Reply by stratofax** [reply]
I requested access to the Google Drive doc you linked to, above.
